src/components/BaseInput.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="field-container">
+    <label v-if="label" class="label">{{ label }}</label>
+    <input
+      v-bind="$attrs"
+      :value="modelValue"
+      :placeholder="label"
+      autocomplete="off"
+      @input="$emit('update:modelValue', $event.target.value)"
+      class="field"
+    />
+  </div>
+</template>
+
+<script>
```

Comment:
Why don't you use typescript here?

Reply:
I think that it doesn't add value to component